### ISSUE TITLE
ERROR:root:/dev/video0 H264 format not available

### ISSUE THREAD

**Issue by miguelwon** [soyersoyer/fmp4streamer#9]
Hi,
I'm getting a `H264 format not available` error but I'm using C930e Logitech camera, that supports H264 encoding.

**Comment by soyersoyer** [comment]
Hi Miguel,
The C930e puts the H264 into the MJPG. Try with the MJPGH264 format.

**Comment by miguelwon** [comment]
Ok, thanks. It works but with very high latency.

**Comment by soyersoyer** [comment]
Do you have this line in your config?
`uvcx_h264_i_frame_period = 1000`

**Comment by miguelwon** [comment]
Yes. I tried to lower it (to 900 e.g.) but when I do it freezes the script in step `Waiting for the first h264 frame...`

**Comment by soyersoyer** [comment]
I think the minimum is 500 and you can increase by 500. But it only means 500ms. With Firefox the latency is better.
